### PR TITLE
Changelog v1.14.3

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Comment by 6543** [comment]
just blocked by #16068 and it's backport

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,47 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.14.3](https://github.com/go-gitea/gitea/releases/tag/v1.14.3) - 2021-06-10
+
+* API
+  * Fix overly strict edit pr permissions (#15900) (#16081)
+* BUGFIXES
+  * Fix language switch for install page (#16043) (#16128)
+  * Fix bug on getIssueIDsByRepoID (#16119) (#16124)
```
Comment (6543) [inline CHANGELOG.md:22]:
```suggestion
  * Fix bug on getIssueIDsByRepoID (#16119) (#16124)
  * Set self-adjusting deadline for connection writing (#16068) (#16123)
```

**Review by 6543 — COMMENTED** [review]
ready from my side :rocket:

**Comment by tomaswarynyca** [comment]
Could #16130 be fixed before releasing the new version?

**Review by techknowlogick — DISMISSED** [review]
Blocking per conversation in discord.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,48 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix truncate utf8 string (#15828) (#15854)
+  * Fix bound address/port for caddy's certmagic library (#15758) (#15848)
+  * Upgrade unrolled/render to v1.1.1 (#15845) (#15846)
+  * Queue manager FlushAll can loop rapidly - add delay (#15733) (#15840)
+  * Tagger can be empty, as can Commit and Author - tolerate this (#15835) (#15839)
+  * Set autocomplete off on branches selector (#15809) (#15833)
+  * Add missing error to Doctor log (#15813) (#15824)
+  * Move restore repo to internal router and invoke from command to avoid open the same db file or queues files (#15790) (#15816)
+* ENHANCEMENTS
+  * Move sans-serif fallback font higher than emoji fonts (#15855) (#15892)
```
Comment (6543) [inline CHANGELOG.md:52]:
```suggestion
  * Removable media support to snap package (#16136) (#16138)
  * Move sans-serif fallback font higher than emoji fonts (#15855) (#15892)
```

**Comment by bminer** [comment]
Just want to ensure #15859 ends up in the Changelog (I saw #15758, but this PR will not work without the other one). Did #15859 get backported?

**Review by khmarbaise — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,56 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.14.3](https://github.com/go-gitea/gitea/releases/tag/v1.14.3) - 2021-06-10
+
+* SECURITY
+  * Encrypt migration credentials at rest (#15895) (#16187)
```
Comment (zeripath) [inline CHANGELOG.md:10]:
```suggestion
  * Encrypt migration credentials at rest (#15895) (#16187)
  * Only check access tokens if they are likely to be tokens (#16164) (#16171)
  * Add missing SameSite settings for the i_like_gitea cookie (#16037) (#16039)
  * Fix setting of SameSite on cookies (#15989) (#15991)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,56 @@ This changelog goes through all the changes that have been made in each release
...
+* API
+  * Repository object only count releases as releases (#16184) (#16190)
+  * EditOrg respect RepoAdminChangeTeamAccess option (#16184) (#16190)
+  * Fix overly strict edit pr permissions (#15900) (#16081)
+* BUGFIXES
+  * Run processors on whole of text (#16155) (#16185)
+  * Class `issue-keyword` is being incorrectly stripped off spans (#16163) (#16172)
+  * Only check access tokens if they are likely to be tokens (#16164) (#16171)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,56 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix language switch for install page (#16043) (#16128)
+  * Fix bug on getIssueIDsByRepoID (#16119) (#16124)
+  * Set self-adjusting deadline for connection writing (#16068) (#16123)
+  * Fix http path bug (#16117) (#16120)
+  * Fix data URI scramble (#16098) (#16118)
+  * Merge all deleteBranch as one function and also fix bug when delete branch don't close related PRs (#16067) (#16097)
+  * git migration: don't prompt interactively for clone credentials (#15902) (#16082)
+  * Fix case change in ownernames (#16045) (#16050)
+  * Add missing SameSite settings for the i_like_gitea cookie (#16037) (#16039)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,56 @@ This changelog goes through all the changes that have been made in each release
...
+  * Don't manipulate input params in email notification (#16011) (#16033)
+  * Fix setting of SameSite on cookies (#15989) (#15991)
```
Comment (zeripath) [inline CHANGELOG.md]:
```suggestion
```

**Review by zeripath — APPROVED** [review]

**Comment by 6543** [comment]
-> #16196

**Pull request merged**